roofit/roofitcore/inc/RooAddPdf.h

```diff
@@ -65,6 +65,8 @@ class RooAddPdf : public RooAbsPdf {
   /// is the sum of all coefficients.
   double expectedEvents(const RooArgSet* nset) const override;
 
+  std::unique_ptr<RooAbsReal> createExpectedEvents(const RooArgSet* nset) const override;
```

Comment:
I don't like too much the name, it looks like we are creating the events, but I am not sure how would call it, maybe `getExpectedEvents` or `createExpectedEventsFunc`

Reply:
You are right, `createExpectedEventsFunc` is better. I wanted to have `create` in there, so it's analogous to `createNLL` or `createIntegral` and all the other functions that create new RooAbsArgs.